From 875130ee3cbcf62266901ef4b91b9a0fb6adad43 Mon Sep 17 00:00:00 2001
From: Tomek Bury <tomek.bury@broadcom.com>
Date: Fri, 4 Aug 2017 16:16:38 +0100
Subject: Use sysroot prefix for pkgdatadir variable

The pc_sysroot is automatically added to cflags and libs but not
to 'pkg-config --variable'

Upstream-Status: Backport [https://cgit.freedesktop.org/wayland/wayland-protocols/commit/?id=875130ee3cbcf62266901ef4b91b9a0fb6adad43]

Reviewed-by: Daniel Stone <daniels@collabora.com>

diff --git a/wayland-protocols.pc.in b/wayland-protocols.pc.in
index a26744c..379be06 100644
--- a/wayland-protocols.pc.in
+++ b/wayland-protocols.pc.in
@@ -1,6 +1,6 @@
 prefix=@prefix@
 datarootdir=@datarootdir@
-pkgdatadir=@datadir@/@PACKAGE@
+pkgdatadir=${pc_sysrootdir}@datadir@/@PACKAGE@
 
 Name: Wayland Protocols
 Description: Wayland protocol files
-- 
cgit v0.10.2

